### ISSUE TITLE
Velocity shim using deprecated Ember.$()

### ISSUE THREAD

**Issue by gitKrystan** [ember-animation/liquid-fire#632]
I'm working on removing `Ember.$()` usages from our app and ran into this warning on Ember canary:

> DEPRECATION: Using Ember.$() has been deprecated, use `import jQuery from 'jquery';` instead

from this line:

https://github.com/ember-animation/liquid-fire/blob/21711359faf0a396489f169ae34c1637e7f45828/vendor/shims/velocity.js#L6

It appears that `self.Velocity` is undefined in our app, so it's falling back to `self.Ember.$`. If I add
```
app.import('node_modules/velocity-animate/velocity.js', { prepend: true });
```
to our build, I no longer get the warning.

**Comment by ef4** [comment]
That is not the only issue, liquid-fire itself depends on jQuery and leaks jQuery into its public API. We're going to need to do a breaking major to get rid of it. I'm assuming that will also be the breaking major where liquid-fire switches to delegating most work to ember-animated.

**Comment by ef4** [comment]
Oh, I might have read too fast. We can clear this particular deprecation without removing jQuery entirely.

**Comment by gabrielgrant** [comment]
Any update on this? liquid-fire seems to be breaking completely in Octane because of this (though realize fixing liquid-fire in Octane properly may well involve the complete transition/breaking change mentioned above)

**Comment by ef4** [comment]
liquid-fire absolutely works in octane. Octane is allowed to still use jQuery, it's just not required or installed by default. You need to add `@ember/jquery` and enable the jquery-integration optional feature.

**Comment by alexlafroscia** [comment]
Would it be possible to change the shim so that it imports `jQuery` as a model, rather than accessing the reference that hangs off of `Ember` ?

**Comment by ef4** [comment]
Yes, somebody should please do that.

**Comment by alexlafroscia** [comment]
I'll take a look at doing that soon -- we have a ticket to clean up those jQuery deprecations and I think this might be the last one!

**Comment by jherdman** [comment]
I believe this is closed as of d6c0a55544e0367d132b43a49a58069ed07b2d1a

**Comment by gabrielgrant** [comment]
looks like "someone" finally got around to doing this ;)

(but seriously, thank you _so, so much_ for all your work on this, and everything else you do to help keep the Ember ecosystem ticking, @ef4 ❤️ ❤️  )
